build/DependencyVersions.props

```diff
@@ -4,15 +4,15 @@
     <DotNetCoreSdkLKGVersion>3.0.100-preview1-009020</DotNetCoreSdkLKGVersion>
     <MicrosoftAspNetCoreAllPackageVersion>2.2.0-preview1-34355</MicrosoftAspNetCoreAllPackageVersion>
     <MicrosoftAspNetCoreAppPackageVersion>$(MicrosoftAspNetCoreAllPackageVersion)</MicrosoftAspNetCoreAppPackageVersion>
-    <MicrosoftNETCoreAppPackageVersion>2.2.0-preview1-26620-03</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview1-26710-03</MicrosoftNETCoreAppPackageVersion>
```

Comment:
I'm actually surprised that we have this version in this repo after the repo split. I would imagine we can get down to just picking up LKG from core-sdk and just specifying TFM. We may even want to consider not retargeting dotnet.dll to bleeding edge TFM unless we actually need to.